examples/ota-requestor-app/efr32/src/AppTask.cpp

```diff
@@ -511,3 +526,23 @@ void AppTask::DispatchEvent(AppEvent * aEvent)
 }
 
 void AppTask::UpdateClusterState(void) {}
+
+void AppTask::InitOTARequestor()
+{
+
+    // Initialize and interconnect the Requestor and Image Processor objects -- START
+    SetRequestorInstance(&gRequestorCore);
+
+    gRequestorCore.Init(&(chip::Server::GetInstance()), &gRequestorUser, &gDownloader);
+
+    gRequestorUser.Init(&gRequestorCore, &gImageProcessor);
+
+    OTAImageProcessorParams ipParams;
+    ipParams.imageFile = CharSpan("test.txt");
```

Comment:
Would have been nice to have this file name pass as a parameter to InitOTARequestor(myFile)

Reply:
This is an outdated API anyway. There is a ticket to fix it in the core OTA logic: https://github.com/project-chip/connectedhomeip/issues/12600